honssh/honeypot/honeypot-docker.py

```diff
@@ -50,6 +50,7 @@ def __init__(self):
         self.container = None
         self.sensor_name = None
         self.peer_ip = None
+        self.channelOpen = False
```

Comment:
You made loads of changes from thisThat to this_that, and now channelOpen? :smile: